src/utils/i18n.ts

```diff
@@ -1,17 +1,21 @@
+// @ts-ignore
```

Comment:
In a ``modules.d.ts``:

```ts
declare module "@dhis2/d2-i18n" {
    export function t(value: string): string;
    export function t(value: string, options?: {
        nsSeparator?: boolean;
        [key: string]: any;
    }): string;
}
```

Reply:
Done. Thanks @SferaDev, I was kind of lazy :)